### DISCUSSION TITLE
WiFi Power saving mode not working

### DISCUSSION THREAD

**Discussion by case485** [micropython/micropython#11903] [General]
Hello, I am trying to disable powersaving mode on wifi but it doesn't seem to exist in the WLAN function as described. 

import network 
sta = network.WLAN(network.STA_IF)
sta.active(True)
sta.connect("West3")
sta.config(pm=PM_NONE)
sta.ifconfg()

The error I receive is : 
Traceback (most recent call last):
  File "<stdin>", line 5, in <module>
NameError: name 'PM_NONE' isn't defined

Board Firmware Info: 
MicroPython v1.20.0 on 2023-04-26; ESP32 module with ESP32

**Comment by felixdoerre** [comment]
Have you tried this?

`sta.config(pm=sta.PM_NONE)`

**Reply by case485** [reply]
Get the same error : 
import network 
sta = network.WLAN(network.STA_IF)
sta.active(True)
sta.connect("West3")
sta.config(pm=sta.PM_NONE)
print(sta.ifconfg())

Error: 
Traceback (most recent call last):
  File "<stdin>", line 5, in <module>
AttributeError: 'WLAN' object has no attribute 'PM_NONE'

**Reply by glenn20** [reply]
The PR adding the `pm` option (#8993) was only merged after 1.20 was released. You need to use a nightly build from after May 6 2023 (or compile your own image).

**Reply by ProfessorKaos** [reply]
Thanks @felixdoerre, this sorted it out for me
